app/src/main/java/com/vci/vectorcamapp/settings/presentation/components/SettingsInfoTile.kt

```diff
@@ -0,0 +1,102 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.rotate
+import androidx.compose.ui.res.painterResource
+import com.vci.vectorcamapp.R
+import com.vci.vectorcamapp.core.presentation.components.tile.ActionTile
+import com.vci.vectorcamapp.ui.extensions.colors
+import com.vci.vectorcamapp.ui.extensions.dimensions
+
+@Composable
+fun SettingsInfoTile(
+    title: String,
+    modifier: Modifier = Modifier,
+    initiallyExpanded: Boolean = false,
+    content: (@Composable ColumnScope.() -> Unit),
+) {
+    var isExpanded by remember { mutableStateOf(initiallyExpanded) }
+
+    val chevronRotation by animateFloatAsState(
+        targetValue = if (isExpanded) -180f else 0f,
+        label = "chevron_rotation"
+    )
+
+    val tileContent: @Composable () -> Unit = {
+        Column(
+            verticalArrangement = Arrangement.spacedBy(MaterialTheme.dimensions.spacingMedium),
+            modifier = Modifier.padding(MaterialTheme.dimensions.paddingLarge)
+        ) {
+            Row(
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                Text(
+                    text = title,
+                    style = MaterialTheme.typography.headlineSmall,
+                    color = MaterialTheme.colors.textPrimary
+                )
+
+
+                Box(
+                    contentAlignment = Alignment.Center,
+                    modifier = Modifier
+                        .size(MaterialTheme.dimensions.componentHeightSmall)
+                        .background(
+                            color = MaterialTheme.colors.iconBackground,
+                            shape = CircleShape
+                        )
+                ) {
+                    Icon(
+                        painter = painterResource(R.drawable.ic_arrow_down),
+                        contentDescription = if (isExpanded) "Collapse" else "Expand",
+                        tint = MaterialTheme.colors.icon,
+                        modifier = Modifier
+                            .size(MaterialTheme.dimensions.iconSizeMedium)
+                            .rotate(chevronRotation)
+                    )
+                }
+            }
+
+            AnimatedVisibility(
+                visible = isExpanded,
+                enter = expandVertically(),
+                exit = shrinkVertically()
+            ) {
+                Column {
+                    content()
+                }
+            }
+        }
+    }
+
+    ActionTile(
+        onClick = { isExpanded = !isExpanded },
+        modifier = modifier,
+        content = tileContent
```

Comment:
Same as above, you can render stuff inside the action tile component itself:

ActionTile(...) {
     // Render content here
}